### PR TITLE
Issue#32

### PR DESCRIPTION
**Pull request by Mellorison**
Fixed EDI Button bug on Patient Dashboard

### CODE REVIEW

**Comment by Mellorison** [comment]
Resolved build Conflicts between issue#32 and develop branch

File: cervicalcancer/src/main/java/zm/gov/moh/cervicalcancer/submodule/dashboard/patient/view/PatientDashboardEDIGalleryFragment.java [hunk 1]
```diff
@@ -117,49 +127,71 @@ public void onBindViewHolder(ViewHolder viewHolder, int i) {
...
+
+                private WebView touch;
+                private Dialog currentAnimator;
+                private boolean isImageFitToScreen;
+                private File file;
+
+                @Override
+                public void onClick(View v) {
+
+                    /*Intent intent = new Intent();
```
Comment (hallix) [inline cervicalcancer/src/main/java/zm/gov/moh/cervicalcancer/submodule/dashboard/patient/view/PatientDashboardEDIGalleryFragment.java]: Remove commented code or add a TODO comment if meant for later use.

File: common/src/main/java/zm/gov/moh/common/submodule/form/widget/DatePickerWidget.java [hunk 2]
```diff
@@ -27,6 +27,7 @@ public void onCreateView() {
         super.onCreateView();
         AppCompatImageButton button = new AppCompatImageButton(mContext);
         super.setGravity(Gravity.CENTER_HORIZONTAL);
+        //mEdittext.setEnabled(false);
```
Comment (hallix) [inline common/src/main/java/zm/gov/moh/common/submodule/form/widget/DatePickerWidget.java]: Remove commented code or a TODO comment if meant for later use.

File: common/src/main/java/zm/gov/moh/common/submodule/form/widget/FormImageViewButtonWidget.java [hunk 3]
```diff
@@ -106,8 +106,6 @@ public void onClick(View v) {
         mBundle.putString(Key.VIEW_TAG, (String)getTag());
         //Toast.makeText(FormImageViewButtonWidget.this, "You clicked on ImageView", Toast.LENGTH_LONG).show();
```
Comment (hallix) [inline common/src/main/java/zm/gov/moh/common/submodule/form/widget/FormImageViewButtonWidget.java]: Remove commented code or a TODO comment if meant for later use.

File: common/src/main/java/zm/gov/moh/common/submodule/login/view/LoginActivity.java [hunk 4]
```diff
@@ -135,6 +136,19 @@ public void setLocation(List<Location> locations){
         locationArrayAdapter.notifyDataSetChanged();
     }
 
+    boolean doubleBackToExitPressedOnce = false;
+   /* private Toast exitToast;
```
Comment (hallix) [inline common/src/main/java/zm/gov/moh/common/submodule/login/view/LoginActivity.java]: Remove commented code or a TODO comment if meant for later use.

**Comment by Mellorison** [comment]
Pushed TODO For highlighted tasks and readded imageview expand method on EDI Class

**Comment by Mellorison** [comment]
Made date fields in all Date-picker text fields read-only.

**Pull request merged**